ext_conf_template.txt

```diff
@@ -2,7 +2,7 @@
 url =
 
 # cat=basic/API; type=string; label=OnTheFly derivative's Url (optional);(https://help.bynder.com/Modules/Asset-Bank/Modify-public-derivatives-on-the-fly.htm?Highlight=on-the-fly#prereq)
-otfBaseUrl =
+otf_base_url =
```

Comment:
Dit heb ik gelijk getrokken met overige key=>values uit dit bestand

Reply:
Maybe add a little note about this in the readme. Because now when you open the extension settings the field will be empty if you update from a previous version. _Or maybe create a upgrade wizard for this :)_